src/layout/section/faqSection/FaqSection.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { Fragment } from 'react'
+
+import { Button } from '../../../button'
+import { ContentDivider } from '../../../divider/contentDivider'
+import { SpacingDivider, SpacingDividerSize } from '../../../divider/spacingDivider'
+import { Column } from '../../../layout/column'
+import { Columns } from '../../../layout/columns'
+import { SectionContentSize } from '../../../layout/section/baseSection'
+import { Paragraph } from '../../../paragraph'
+import { StyledFaqSection } from './FaqSection.style'
+
+export type FaqItemProps = {
+  question: string
+  answer: string
+}
+
+export type FaqSectionProps = Readonly<{
+  className?: string
+  title?: string
+  items: Array<FaqItemProps> // max 6 FAQ items
+  expandLabel: string
+  buttonLabel?: string
+  buttonHref?: string | JSX.Element
+}>
+
+/**
+ * A specialized section with up to 6 expandable paragraphs with Questions & Answers.
+ *
+ * Note: use of Columns is not ideal as it creates a listitem for each column
+ * We would rather have one listitem by FAQ item instead
+ */
+export const FaqSection = (props: FaqSectionProps) => {
+  const { className, items, title, buttonLabel, buttonHref, expandLabel } = props
+
+  const renderFaqItem = ({ question, answer }: FaqItemProps): JSX.Element => (
+    <Fragment>
+      <StyledFaqSection.Question as="h3">{question}</StyledFaqSection.Question>
+      <Paragraph isExpandable expandLabel={expandLabel}>
+        {answer}
+      </Paragraph>
+      <ContentDivider />
+      <SpacingDivider size={SpacingDividerSize.SMALL} />
+    </Fragment>
+  )
+
+  return (
+    <StyledFaqSection.Section
+      contentSize={SectionContentSize.LARGE}
+      tagName="article"
+      noHorizontalSpacing
+      className={className}
+    >
+      {title && <StyledFaqSection.Title as="h2">{title}</StyledFaqSection.Title>}
+
+      <Columns>
+        <Column>
```

Comment:
You might be able to write a more flexible code for this by splitting the items into two arrays (pair/impair) then render the columns. You won't have to check for each item, and we'll be able to easilly add more items if we want to.

Reply:
I changed it, instead of going with <Columns> (as it was when I took it from the /pro page at first) I simply used a list with a couple of rules to split it in 2 when viewing on a large display. No more limit and a much more satisfying list as a result.